src/neuroglancer/datasource/dvid/base.ts

```diff
@@ -21,6 +22,7 @@ export class DVIDSourceParameters {
 }
 
 export class VolumeChunkSourceParameters extends DVIDSourceParameters {
+  volumeType: VolumeType;
```

Comment:
I think it would be better to use an explicit encoding enum, e.g.:

export enum VolumeChunkEncoding { JPEG, COMPRESSED_SEGMENTATION }

and then add encoding: VolumeChunkEncoding as a parameter.  That makes it easier to support other encodings in the future.